### ISSUE TITLE
Audit Formation color tokens for adding to the CSS Library

### ISSUE THREAD

**Issue by jamigibbs** [department-of-veterans-affairs/vets-design-system-documentation#2294]
## Description
While we were updating the CSS Library for the [USWDS blue colors](https://github.com/department-of-veterans-affairs/vets-website/pull/26724), we noticed that some of the tokens updated in Formation did not exist yet in the CSS Library.

For example:
- color-aqua
- color-aqua-dark
- color-aqua-lighest

The purpose of this issue is to audit the [Formation color tokens](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/_b-variables.scss) and check to see if there are any that we want to move to the CSS Library (like the 3 above).

Questions to ask for each color:
1. Is this color still being used?
2. If yes, do we still want it to be in the CSS Library?

## Tasks
- [ ] Formation color tokens have been checked to see if there are any that need to be moved to the CSS Library
- [ ] The CSS Library has been updated with those tokens

## Acceptance Criteria
- [ ] Formation audited and CSS Library updated
- [ ] CSS Library version incremented and a new release published

**Comment by humancompanion-usds** [comment]
There are a number of colors that should have been deprecated ages ago and I'm loath to copy them into the clean slate of the new CSS library. We made the decision to not offer up the primitive colors in Figma thus forcing designers to use semantic and component tokens. I don't want developers circumventing that decision by being able to use primitive tokens. But I understand we likely have many of those in use. Thus there are categories of the primitives that I'd like to see us eliminate and color-aqua is definitely on that list. 

I am also concerned that very few of the new semantic tokens make it into CSS library. They really all need to be there as we've provided them in Figma already. https://design.va.gov/foundation/color-palette#semantic-color-tokens
